lib/js/helpers/classy.js

```diff
@@ -14,7 +14,7 @@ export function has (domNode, className) {
 export function add (domNode, classes) {
   classes.split(' ').forEach(function (c) {
     if (!has(domNode, c)) {
-      var existingClass = domNode.getAttribute('class') || "";
+      var existingClass = domNode.getAttribute('class') || '';
```

Comment:
Yeah, good idea. I'll open an issue and we can address that before moving to v1.0